.github/workflows/qa.yml

```diff
@@ -0,0 +1,28 @@
+name: QA
+
+on:
+    pull_request:
+        branches:
+            - master
+
+jobs:
+    qa:
+        runs-on: ubuntu-18.04
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: actions/setup-node@v1
+                with:
+                    always-auth: true
+                    node-version: '12'
```

Comment:
Moet dit nog op meerdere versies worden getest evt.?

Reply:
Dat zou een mogelijke verbetering zijn, ik kan dat alleen zelf niet beoordelen. Het is voor nu een een-op-een vervanging van de job die in Jenkins stond. Dat kan later door een maintainer worden opgepakt.